go/BUILD.bazel

```diff
@@ -44,50 +43,27 @@ pkg_files(
     ],
 )
 
-pkg_filegroup(
-    name = "extractor-pack-generic",
-    srcs = [
-        ":resources",
-        "//go/codeql-tools",
-        "//go/downgrades",
-        "//go/extractor:tokenizer",
-    ],
-    visibility = ["//visibility:public"],
-)
-
-pkg_files(
+codeql_pkg_files(
     name = "extractor-pack-arch",
-    srcs = [
+    exes = [
         "//go/extractor/cli/go-autobuilder",
         "//go/extractor/cli/go-bootstrap",
         "//go/extractor/cli/go-build-runner",
         "//go/extractor/cli/go-extractor",
         "//go/extractor/cli/go-gen-dbscheme",
         "//go/extractor/cli/go-tokenizer",
     ],
-    attributes = pkg_attributes(mode = "0755"),
-    prefix = "tools/" + codeql_platform,
-    visibility = ["//visibility:public"],
+    prefix = "tools/{CODEQL_PLATFORM}",
 )
 
-pkg_filegroup(
-    name = "extractor-pack",
+codeql_pack(
+    name = "go",
     srcs = [
         ":extractor-pack-arch",
-        ":extractor-pack-generic",
+        ":resources",
+        "//go/codeql-tools",
+        "//go/downgrades",
+        "//go/extractor:tokenizer",
     ],
-    visibility = ["//visibility:public"],
-)
-
-pkg_install(
-    name = "_extractor_pack",
-    srcs = [":extractor-pack"],
-)
-
-py_binary(
-    name = "create-extractor-pack",
-    srcs = ["create_extractor_pack.py"],
-    env = {"REPO_NAME": repo_name()},
-    main = "create_extractor_pack.py",
-    deps = ["_extractor_pack"],
+    install_dest = "build/codeql-extractor-pack",
```

Comment:
@mbg are you particularly attached to this installation destination? If we can go with the default `go/extractor-pack` it would be consistent with other languages already using that (`ruby`, `ql` and `swift`), and we could go for a generic `*/extractor-pack/qlpack.yml` include in `codeql-workspace.yml` and a generic `/*/extractor-pack` exclude in `.gitignore`.

Reply:
I am not aware of any reason why this has to be `codeql-extractor-pack`. I suspect it may just be an inconsistency from when the Go extractor lived in a separate repository. It should be fine to change this to be the same as for other languages. CC @smowton @owen-mc in case they have any insights I don't have.